### DISCUSSION TITLE
Sync Issues on Raspian64-ARM GUI

### DISCUSSION THREAD

**Discussion by arlennp** [Chia-Network/chia-blockchain#9234] [Support]
The Full Node is synced and offers plotting or farming, but the wallet is not synced. It is stuck on block 231,115 and as such does not recognize balances or Pools.

**Comment by loppefaaret** [comment]
coud you try and locate your `debug.log` file, under you users folder `~/.chia/mainnet/log/` and attach it to this issue

**Comment by arlennp** [comment]
[debug.log](https://github.com/Chia-Network/chia-blockchain/files/7506033/debug.log)

**Comment by loppefaaret** [comment]
`2021-11-06T19:09:49.375 wallet chia.wallet.wallet_blockchain: ERROR    Error during db transaction: database disk image is malformed`
im sorry to inform you, but your wallet database is corrupted, and needs to be move out, and started from new:
LINUX WALLET DATABASE RESET

Close down Chia, and open up a terminal, and execute the below command:
```
mv ~/.chia/mainnet/wallet/db/block* ~/Desktop/
```
Then try opening up the Chia again
if it worked, you can remove the files we put on the Desktop!

**Comment by paulhainsworth-chia** [comment]
Can we please look at the log to determine if we can find root cause for this defect that we need to address?

**Reply by arlennp** [reply]
Sorry, I may not be able to share it.
The above solution looked like it was working. The wallet started syncing again, but then stalled around 35000 with similar text in the debug file.  When I tried the same solution again, the program would no longer load fully. After uninstalling and reinstalling, Chia blockchain would still not fully load. 
I reformatted the card, flashed the 64bit OS again, reinstalled Chia GUI. It is now syncing and around block 49000. 
If the issue recurs, I will be happy to share the log.
